staging/apps/spark-jobs/fhir-export-etl/job.yml

```diff
@@ -0,0 +1,53 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: fhir-export
+spec:
+  template:
+    spec:
+      containers:
+        - name: fhir-export
+          image: ferlabcrsj/clin-pipelines
+          args:
+            - bio.ferlab.clin.etl.FhirExport
+            - all
+          env:
+            - name: FHIR_URL
+              value: https://fhir.staging.clin.ferlab.bio/fhir
+            - name: KEYCLOAK_URL
+              value: https://auth.staging.clin.ferlab.bio/auth
+            - name: KEYCLOAK_CLIENT_KEY
+              value: clin-system
+            - name: KEYCLOAK_CLIENT_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: keycloak-clin-system-client-credentials
+                  key: KEYCLOAK_CLIENT_SECRET
+            - name: KEYCLOAK_AUTHORIZATION_AUDIENCE
+              value: clin-acl
+            - name: AWS_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: s3-etl-credentials
+                  key: S3_ACCESS_KEY
+            - name: AWS_SECRET_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: s3-etl-credentials
+                  key: S3_SECRET_KEY
+            - name: AWS_ENDPOINT
+              value: https://s3.ops.cqdg.ferlab.bio
+            - name: AWS_DEFAULT_REGION
+              value: regionone
+            - name: AWS_BUCKET_NAME
+              value: clin-staging-app-datalake
+            - name: AWS_PREFIX
+              value: $COLOR
```

Comment:
TODO

Reply:
This env var is not used in this job. You can remove it